install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -146,6 +146,10 @@
           optional: true
       - name: uds-socket
         emptyDir: {}
+      - name: telemetry-adapter-secret
```

Comment:
we might want to think about having this be in the policy deployment as well, just in case.

Reply:
Done.